controls/chat/styling.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Styling
+page_title: .NET MAUI Chat Documentation - Styling
+description: Learn how to style the Telerik UI for .NET MAUI Chat control.
+position: 6
+tags: .net maui, telerik .net maui, ui for .net maui, chat, style
+slug: chat-styling
+---
+
+# .NET MAUI Chat Styling
+
+Any change in the appearance of the `RadChat` components depends on the referenced styles. The default templates contains a `RadBorder` control (used to achieve the rounded edges), an Image control (used for the avatar image only for the single and first messages) and Label for the text message itself.
+
+Here are the provided styles that you can use to customize the different parts of the control:
+
+- `MessageImageStyle`(`TargetType Image`)&mdash;Defines the style referred to the send button image.
```

Comment:
`MessageImageStyle` (target type `Image`)&mdash;Defines the style of the outgoing message image.

same applicable for all properties below. also you have to explain that all styles available for the image can be applied to the MessageImageStyle.